lasagne/layers/conv.py

```diff
@@ -1127,8 +1126,16 @@ def convolve(self, input, **kwargs):
         conved = op(self.W, input, output_size)
         return conved
 
+
 Deconv3DLayer = TransposedConv3DLayer
 
+if not hasattr(T.nnet.abstract_conv, 'AbstractConv3d_gradInputs'):
+    # pragma: no cover
```

Comment:
I put that in the next line because I was not sure whether the pep8 tests will fail (long line). Changed it back to inline